components/UpdatesWithData.js

```diff
@@ -41,8 +45,15 @@ class UpdatesWithData extends React.Component {
     }
   }
 
+  updateQuery = (router, newParams) => {
```

Comment:
:ice_cream: I think it would be better to keep this component agnostic about routes, and handle this in `pages/updates.js`

Reply:
Thanks. Done. 👍🏽